static/app/components/devtoolbar/components/replay/replayPanel.tsx

```diff
@@ -0,0 +1,111 @@
+import {useContext} from 'react';
+import {css} from '@emotion/react';
+
+import {Button} from 'sentry/components/button';
+import AnalyticsProvider, {
+  AnalyticsContext,
+} from 'sentry/components/devtoolbar/components/analyticsProvider';
+import SentryAppLink from 'sentry/components/devtoolbar/components/sentryAppLink';
+import useReplayRecorder from 'sentry/components/devtoolbar/hooks/useReplayRecorder';
+import {resetFlexRowCss} from 'sentry/components/devtoolbar/styles/reset';
+import ProjectBadge from 'sentry/components/idBadge/projectBadge';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {IconPause, IconPlay} from 'sentry/icons';
+import type {PlatformKey} from 'sentry/types/project';
+
+import useConfiguration from '../../hooks/useConfiguration';
+import {panelInsetContentCss, panelSectionCss} from '../../styles/panel';
+import {smallCss} from '../../styles/typography';
+import PanelLayout from '../panelLayout';
+
+const TRUNC_ID_LENGTH = 16;
+
+export default function ReplayPanel() {
+  const {projectSlug, projectId, projectPlatform, trackAnalytics} = useConfiguration();
+
+  const {disabledReason, isDisabled, isRecording, lastReplayId, start, stop} =
+    useReplayRecorder();
+
+  function ReplayLink({children}: {children: React.ReactNode}) {
+    const {eventName, eventKey} = useContext(AnalyticsContext);
+    return process.env.NODE_ENV === 'production' ? (
+      <SentryAppLink
+        to={{
+          url: `/replays/${lastReplayId}`,
+          query: {project: projectId},
+        }}
+      >
+        {children}
+      </SentryAppLink>
+    ) : (
+      <ExternalLink
+        href={`https://sentry-test.sentry.io/replays/${lastReplayId}/?project=5270453`}
+        onClick={() => {
+          trackAnalytics?.({
+            eventKey: eventKey + '.click',
+            eventName: eventName + ' clicked',
+          });
+        }}
+      >
+        {children}
+      </ExternalLink>
+    );
+  }
+
+  return (
+    <PanelLayout title="Session Replay">
+      <AnalyticsProvider
+        keyVal={`replay-button-${isRecording ? 'stop' : 'start'}`}
+        nameVal={`replay button ${isRecording ? 'stop' : 'start'}`}
+      >
```

Comment:
Whether the replay is started or stopped is more of a property, the button is the toggle button
```suggestion
      <AnalyticsProvider
        keyVal="toggle-replay-button"
        nameVal="toggle replay button"
      >
```